controlpanel/frontend/jinja2/user-detail.html

```diff
@@ -99,8 +99,8 @@ <h2 class="govuk-heading-m">{{ pronoun }} webapp data sources</h2>
 <section class="cpanel-section">
   <form action="{{ url("delete-user", kwargs={ "pk": user.auth0_id }) }}" method="post">
     {{ csrf_input }}
-    <button class="govuk-button cpanel-button--destructive confirm"
-            data-confirmation-prompt="Are you sure you want to delete this user?">
+    <button class="govuk-button cpanel-button--destructive js-confirm"
```

Comment:
On the other button we made sure it was input submits rather than button...

Is it better to convert these two?

Reply:
Not sure I follow.